integration/setup/setup.go

```diff
@@ -69,5 +72,19 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
+	// Ensure Chart CRD creation
+	{
+		c := k8scrdclient.Config{
+			K8sExtClient: config.Host.ExtClient(),
+			Logger:       config.Logger,
+		}
+
+		var crdClient *k8scrdclient.CRDClient
+		crdClient, err = k8scrdclient.New(c)
```

Comment:
The client initialization should be done in its own block. This is a bit off. It is usually copy paste so I am not sure what the problem is with the current practise.